recipe/FindVTK.cmake

```diff
@@ -0,0 +1,53 @@
+message( STATUS Using custom FindVTK )
```

Comment:
@adam-urbanczyk is this file from scratch, or did you copy it from somewhere?

Reply:
I wrote it.